### PR TITLE
Fix Icon Loading Issue for PM Blocks

### PR DESCRIPTION
**Pull request by sanjacornelius**
This PR addresses an issue where the Icon Select for PM Blocks was not loading icons. The problem occurred because the Icons.js file was overwritten with icons for the Launchpad, causing conflicts. 

## Solution
To resolve this, the original icons for Icons.js were restored, and a separate file, LaunchpadIcons.js, was created and imported into the icon-dropdown component.

## How to Test

**PM Block Icons**

1. Create a Process.
2. Save the process as a PM Block via the process ellipsis menu.
3. Ensure the Icon Selector loads the proper icons.
4. Save the PM Block.
5. Navigate to PM Blocks.
6. Select 'Configure' in the ellipsis menu.
7. Ensure the proper icon is loaded.
8. Open Modeler.
9. Navigate to the PM Blocks tab.
10. Confirm that PM Block icons are loaded.

**Launch Pad Icons**

1. Navigate to Processes.
2. Select 'Edit in Launchpad'.
3. Ensure Launchpad Icons load properly.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13176

ci:next

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

File: resources/js/components/shared/IconDropdown.vue [hunk 1]
```diff
@@ -220,3 +220,4 @@ $multiselect-height: 38px;
   }
 }
 </style>
+./LaunchpadIcons
```
Comment (fagubla) [inline resources/js/components/shared/IconDropdown.vue:223]: is necesary this line?

File: resources/js/components/shared/IconSelector.vue [hunk 2]
```diff
@@ -283,3 +283,4 @@
     }
   }
 </style>
+./LaunchpadIcons
```
Comment (fagubla) [inline resources/js/components/shared/IconSelector.vue:286]: is necesary this line?

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/components/shared/IconDropdown.vue (hunk 1 above)
Comment (sanjacornelius) [reply resources/js/components/shared/IconDropdown.vue:223]: Oh no didn't notice that good catch!

**Review by mcraeteisha — APPROVED** [review]
Approved!

**Review by fagubla — APPROVED** [review]

**Pull request merged**